core/server/src/main/java/alluxio/worker/block/SpaceReserver.java

```diff
@@ -71,33 +65,6 @@ public SpaceReserver(BlockWorker blockWorker) {
       mBytesToReserveOnTiers.put(tierAlias, reservedSpaceBytes + lastTierReservedBytes);
       lastTierReservedBytes += reservedSpaceBytes;
     }
-    mCheckIntervalMs = Configuration.getInt(Constants.WORKER_TIERED_STORE_RESERVER_INTERVAL_MS);
-    mRunning = true;
-  }
-
-  @Override
-  public void run() {
-    long lastCheckMs = System.currentTimeMillis();
-    while (mRunning) {
-      // Check the time since last check, and wait until it is within check interval
-      long lastIntervalMs = System.currentTimeMillis() - lastCheckMs;
-      long toSleepMs = mCheckIntervalMs - lastIntervalMs;
-      if (toSleepMs > 0) {
-        CommonUtils.sleepMs(LOG, toSleepMs);
-      } else {
-        LOG.warn("Space reserver took: {}, expected: {}", lastIntervalMs, mCheckIntervalMs);
```

Comment:
Does `HeartbeatExecutor` print out messages like this? Would this message be useful?